### PR TITLE
refactor: configs

### PR DESCRIPTION
**Pull request by IgnatovFedor**

### CODE REVIEW

File: deeppavlov/utils/telegram/telegram_ui.py [hunk 1]
```diff
@@ -13,11 +13,11 @@
...
 
 from pathlib import Path
-from typing import Union
+from typing import Optional, Union
 
 from deeppavlov.utils.connector import TelegramBot
 
 
-def interact_model_by_telegram(model_config: Union[str, Path, dict], token: str = None) -> None:
+def interact_model_by_telegram(model_config: Union[str, Path, dict], token: Optional[str]) -> None:
```
Comment (yoptar) [inline deeppavlov/utils/telegram/telegram_ui.py]:
```suggestion
def interact_model_by_telegram(model_config: Union[str, Path, dict], token: str) -> None:
```
or
```suggestion
def interact_model_by_telegram(model_config: Union[str, Path, dict], token: Optional[str] = None) -> None:
```

File: deeppavlov/utils/alexa/server.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
 def start_alexa_server(model_config: Union[str, Path, dict],
                        port: Optional[int] = None,
-                       https: bool = False,
+                       https: Optional[bool] = None,
```
[the remark this answers is not in the document]
Comment (IgnatovFedor) [reply deeppavlov/utils/alexa/server.py]: Cause default `https` argument value in `deep.py` is `None`. If we have not provided this argument, value will be taken from `utils.settings.server_config`

File: deeppavlov/utils/connector/bot.py [hunk 3]
```diff
@@ -139,14 +139,8 @@ def _get_connector_params(self) -> dict:
...
-        bot_defaults = connector_config['bot_defaults'].get(bot_name, {})
-
-        model_name = type(self._model.get_main_component()).__name__
-        models_info = connector_config['models_info']
-        model_msgs = models_info.get(model_name, models_info['@default'])
-        conversation_defaults = connector_config['conversation_defaults']
-        conversation_defaults['start_message'] = model_msgs['start_message']
-        conversation_defaults['help_message'] = model_msgs['help_message']
+        conversation_defaults = connector_config['telegram']
+        bot_defaults = connector_config['deprecated'].get(bot_name, conversation_defaults)
```
[the remark this answers is not in the document]
Comment (IgnatovFedor) [reply deeppavlov/utils/connector/bot.py]: @litinsky requested to move Alexa and MSBot settings to deprecated sections as this connectors will be moved to `dp-agent` and in order to simplify the perception of `server_config` by users from habr.com in second DeepPavlov's article.

File: deeppavlov/utils/server/server.py [hunk 4]
```diff
@@ -64,15 +64,15 @@ def filter(self, record: logging.LogRecord) -> bool:
...
     model_config = parse_config(model_config)
 
     server_params = server_config['common_defaults']
 
-    if check_nested_dict_keys(model_config, ['metadata', 'labels', 'server_utils']):
-        model_tag = model_config['metadata']['labels']['server_utils']
-        if model_tag in server_config['model_defaults']:
+    if check_nested_dict_keys(model_config, ['metadata', 'server_utils']):
+        model_tag = model_config['metadata']['server_utils']
+        if model_tag in server_config.get('model_defaults', {}):
```
[the remark this answers is not in the document]
Comment (IgnatovFedor) [reply deeppavlov/utils/server/server.py]: done (could not commit this suggestion due to typo)

**Pull request merged**